src/components/MovieList.jsx

```diff
@@ -1 +1,19 @@
-// implement MovieList component here
+import React from 'react';
+import PropTypes from 'prop-types';
+import MovieCard from './MovieCard';
+
+class MovieList extends React.Component {
+  render() {
+    const { movies } = this.props;
+    return (
+      <div>
+        { movies.map((movie) => <MovieCard key={ movie.title } movie={ movie } />) }
+      </div>
+    );
+  }
+}
+MovieList.propTypes = {
+  movies: PropTypes.shape([]).isRequired,
```

Comment:
Opa Erick, esta linha provavelmente fara seu console emitir um Warning, acredito que isso ocorra pois o JavaScript não diferencia objetos de array, a não ser que seja forçado. Uma sugestão é utilizar o método arrayOf.

```suggestion
  movies: PropTypes.arrayOf(PropTypes.object).isRequired,
```